keystore/kms/aws/client.go

```diff
@@ -21,72 +22,97 @@ type Configuration struct {
 	Endpoint        *string `json:"endpoint,omitempty"`
 }
 
-// Encryptor implementation of AWS Encryptor
-type Encryptor struct {
+// KMSClient represent general AWS KMS client
+type KMSClient struct {
 	client *kms.Client
 }
 
-// NewEncryptor create new AWS KMS encryptor which implement Encryptor interface
-func NewEncryptor(credentialPath string) (baseKMS.Encryptor, error) {
-	configuration, err := readConfigByPath(credentialPath)
+// NewKmsClient create new KMS AWS client
+func NewKmsClient(cfg *Configuration) (*KMSClient, error) {
```

Comment:
```suggestion
func NewKMSClient(cfg *Configuration) (*KMSClient, error) {
```

It returns KMSClient, so lets name similar